demos/embeddings/requirements.txt

```diff
@@ -0,0 +1,2 @@
+paramiko==2.4.2
```

Comment:
@jessclmoore none of the examples mentioned in the description above require the paramiko package. I ran them in an environment without paramiko installed, they work fine. Are you sure this is required?

Reply:
@youph Perhaps its related to my system. I received the following warning when running demos/embeddings/stellargraph-metapath2vec.ipynb and demos in
node-classification/node2vec/*
link-prediction/random-walks/*

UserWarning: paramiko missing, opening SSH/SCP/SFTP paths will be disabled. pip install paramiko to suppress

Reply:
Weird, I never saw such warnings before. But despite the warnings the demos work for you without `paramiko` installed, right?